### ISSUE TITLE
data/log/wrapper.log since v4.7.3 in Linux release tarball

### ISSUE THREAD

**Issue by herrbischoff** [theotherp/nzbhydra2#812]
I'm in the process of updating the FreeBSD port to the latest version v4.7.4.

During QA, the committer found that the package build fails because a new file appeared in the distribution tarball that was not accounted for. This is specific to the package build and unrelated to NZBHydra.

See: https://bugs.freebsd.org/bugzilla/show_bug.cgi?id=268430

The file in question is `data/log/wrapper.log` with the following content:

```
2022-12-06 17:41:04,265  CRITICAL - Unable to determine base path correctly. Please make sure to run NZBHydra in the folder where its binary is located. Current base path: /mnt/c/Users/strat/IdeaProjects/nzbhydra2/releases/linux-release/include
```

To me, it looks like an accidental artifact probably introduced during release preparation. I'd like to double-check if this file (or entire folder) is intentional or can/should be removed. The folder and contents appeared in v4.7.3.

Thanks for letting me know so we can proceed with updating the port.

**Comment by theotherp** [comment]
Yu're right, that shouldn't be there. I'll remove it in the next release.

**Comment by theotherp** [comment]
Should be fixed in 4.7.6
